packages/vapor/resources/icons/svg/content/disabled-checkbox.svg

```diff
@@ -1 +1 @@
-<svg width='10px' fill='#E5E8E8' viewBox='0 0 10 10' xmlns='http://www.w3.org/2000/svg'><path d='m3.144 8-2.977-3.109c-.223-.232-.223-.61 0-.843s.586-.232.809 0l2.168 2.264 5.881-6.138c.222-.232.585-.232.809 0 .222.233.222.611-.001.844z'/></svg>
\ No newline at end of file
+<svg width='10px' fill='#C4C9CF' viewBox='0 0 10 10' xmlns='http://www.w3.org/2000/svg'><path d='m3.144 8-2.977-3.109c-.223-.232-.223-.61 0-.843s.586-.232.809 0l2.168 2.264 5.881-6.138c.222-.232.585-.232.809 0 .222.233.222.611-.001.844z'/></svg>
```

Comment:
Personally I think this one is still a bit hard to see. Should we check the accessibility of it?

Reply:
![image](https://user-images.githubusercontent.com/63734941/121597743-1163b780-ca0f-11eb-9486-c0d29b4ceb8a.png)
you're right, I did not changed the color for the - :male_detective: 
I'll do it! since the border are grey-50 for all the disabled boxes :+1: